### ISSUE TITLE
Update the peer dependency on @cycle/core

### ISSUE THREAD

**Issue by jessaustin** [cyclejs/cycle-notification-driver#6]
I'm not sure if 3 -> 5 had breaking changes for this module, or I would have sent a PR.

**Comment by secobarbital** [comment]
Would you be willing to try changing the peer dependency to `rx` and updating the code to `import Rx from 'rx'`?

**Comment by jessaustin** [comment]
Sure. I'm a bit apprehensive, however, because I don't seem to be able to test:

``` bash
$ git clone git@github.com:jessaustin/cycle-notification-driver.git
  ...
$ cd cycle-notification-driver
$ npm install
  ...
$ npm test
> @cycle/notification@1.0.0 test /home/jess/dev/cycle-notification-driver
> mocha --compilers js:babel/register test

  0 passing (4ms)
```

I'm sure there's something wrong with my environment (probably a missing global module), but I'm not a mocha expert and I don't see a way forward here. I can skip testing, but I'm not sure how helpful that would be for you.

**Comment by BrandonSmith** [comment]
This should be fixed in the 2.0.0 release as addressed in #8.
